### PR TITLE
Fix import

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: package/yast2-ntp-client.changes [hunk 1]
```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Wed Dec 13 14:40:48 UTC 2017 - jreidinger@suse.com
+
+- Do minimal modification to default chrony.conf in autoyast
+  when chrony is not specified explicitelly in packages section of
```
Comment (schubi2) [inline package/yast2-ntp-client.changes]: I think it is a typo :-) "explicitly"

File: src/modules/NtpClient.rb [hunk 2]
```diff
@@ -409,6 +409,16 @@ def Import(settings)
       true
     end
 
+    # Merges config to existing system configuration. It is useful for delayed write.
+    # When it at first set values, then chrony is installed and then it writes. So
+    # before write it will merge to system. Result is that it keep majority of config
+    # untouched and modify and what is needed.
```
Comment (schubi2) [inline src/modules/NtpClient.rb]: I would remove the last "and"

File: src/modules/NtpClient.rb [hunk 3]
```diff
@@ -409,6 +409,16 @@ def Import(settings)
       true
     end
 
+    # Merges config to existing system configuration. It is useful for delayed write.
+    # When it at first set values, then chrony is installed and then it writes. So
```
Comment (schubi2) [inline src/modules/NtpClient.rb]: Hm, I do not really understand this.

Comment (jreidinger) [reply src/modules/NtpClient.rb]: I try to explain here that if you set values, it works on parsed configuration file, but if package is not yet installed, then it create new configuration file which is missing many stuff. So if package is installed later, then this method re-apply changes on top of now parsed file.

File: src/modules/NtpClient.rb (hunk 2 above)
Comment (jreidinger) [reply src/modules/NtpClient.rb]: :+1:

**Pull request merged**